scripts/contributor-workflow-setup.sh

```diff
@@ -110,6 +110,7 @@ CONTRIBUTOR_PEER=$(rad self --peer)
 cd acme
 
 # Create change
+git checkout -b update-readme
```

Comment:
Does that not break anything?

Reply:
Nope.  Yet I updated it to checkout master afterwards to avoid any future problems.

Reply:
For me it's broken:
```
fatal: commit hashes do not match: 1d09e3c1591bb81d63a78bc86a1a89cb38c2681d vs. c28ebbf6fe1ef53e4908124369f1a531e3f722f6
```

Reply:
What was the reason for changing it?

Reply:
My bad, I didn't notice the error at the end.

The rationale is it being the common workflow to use a feature branch.  The catalyst was the change in this PR to show the branch.

Reply:
As of now, it appears problematic to do this, so I've reverted the commits re: the feature branch.